### PR TITLE
Add Kap as other tool

### PR DESCRIPTION
**Pull request by daxmc99**
Add [kap](https://getkap.co/) as tool
Tagged reviewers who use cool screen recorders

### CODE REVIEW

**Review by rvantonder — APPROVED** [review]
I love Kap

**Comment by rvantonder** [comment]
@mrnugget uses a fancy (paid) screen recorder too, can't remember the name and I haven't tried it yet but want to.

**Comment by emchap** [comment]
I have never heard of Kap but it looks nice! I support its inclusion.

**Review by emchap — APPROVED** [review]

**Comment by mrnugget** [comment]
> @mrnugget uses a fancy (paid) screen recorder too, can't remember the name and I haven't tried it yet but want to.

I use [CleanShot X](https://cleanshot.com/) for screenshots/gifs/short videos. And I use [Screenflow](http://www.telestream.net/screenflow/overview.htm) for longer demo videos in which I also want to show my face (although I think cleanshot might support this by now, because it's awesome)

File: company/remote/teammate-recommended_productivity_hacks.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ You might think of this page as a remote version of "the clever workflows and sh
 
 - [Loom](https://loom.com) for quick videos for demos or bugs.
 - [Textexpander](https://textexpander.com) or [aText](https://www.trankynam.com/atext/) for text expansions and abbreviations. Use it to store things like your Zoom URL, Calendly URL, and our ISO date format in a way that can be easily populated into other documents.
+- [Kap](https://getkap.co/) open source screen recorder for MacOS
```
Comment (mrnugget) [inline company/remote/teammate-recommended_productivity_hacks.md:41]:
```suggestion
- [Kap](https://getkap.co/) open source screen recorder for MacOS 
- [CleanShot X](https://cleanshot.com/) is a paid but incredibly good screenshot/GIF/video  recording tool for macOS. 
- [Screenflow](http://www.telestream.net/screenflow/overview.htm) is a paid video recording product that allows you to record your screen and webcam at the same time.
```

**Review by mrnugget — APPROVED** [review]

**Pull request merged**